config/samples/workloads_v1alpha1_consoletemplate.yaml

```diff
@@ -22,6 +22,7 @@ spec:
       restartPolicy: Never
 metadata:
   name: console-template-0
+  namespace: default
   labels:
-    app: myapp
-    release: myapp-sandbox
+    repo: myapp-owner-myapp-repo
+    env: myapp-env
```

Comment:
We've recently changed our label taxonomy to use `environment` everywhere. What are your thoughts on env vs environment here?

Reply:
Not particularly fussed, will change